README.md

```diff
@@ -2,9 +2,18 @@
 
 This program is written in Python. Its purpose is to collect weather forecast data from a few websites (e.g. Open-Meteo, WeatherAPI) and combine it into one more accurate weather prediction. After providing the location eg. "Madrid" you will receive a file with weather data and a user-friendly plot. The file will contain raw forecast data acquired from web pages and counted mean values. The plot file will contain hourly temperature and precipitation graph.
 
+**Parameters**  
+*Required:*  
+-location - Name of the city/town in English. E.g. 'Madrid', 'Valencia', 'Cracovia’
 
-**Requirements:**
+*Optional:*  
+-forecast_period - Forecast period, with six options to choose from: '24h', '48h', '72h', '7days', '10days', '14days'. (default: 24h)  
+-date - Date for which weather forecast will be presented or will start from, e.g.: '2024-04-01'. (default: current date)  
+-temperature - Two temperature scales to choose from: Celsius [C] and Fahrenheit [F]. (default: C) **!!!not supported for now!!!**  
+-weather_components - Additional weather components to choose from: 'wind', 'pressure', 'both'. **!!!not supported for now!!!**
```

Comment:
Please update two things:
- make the params name formatted with backticks (`) to distinguish them from the other text
- make the notifications about not being supported in a less screamy way, it looks a bit "unprofessional"😛 (e.g. without the/so many exclamation marks and in braces)

Reply:
@KrzysiekJodlowski Done.